lib/item.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './item_price'
+
+class Item
+
+    attr_reader :name, :quantity
+
+    def initialize(item, quantity)
+        @name = item
+        @quantity = quantity
+    end
+
+    def sale_price 
+        ItemPrice.new(@name, @quantity).sale_price
```

Comment:
You could have created a variable item_price in the initialize method and used it here instead.